File: Difficulty: Basic/Last Digit of a number/last-digit-of-a-number.c
//{ Driver Code Starts
//Initial Template for C

#include <stdio.h>


// } Driver Code Ends

// User function Template for C

void utility(int n) {
    // Calculate the absolute value of the last digit
    int ans = abs(n % 10);

    // Print the result
    printf("%d\n", ans);
}


//{ Driver Code Starts.
    
int main() {
	
	int t; scanf("%d", &t);
	while(t--) {
        int n; scanf("%d", &n);
        utility(n);
    }
	
}
// } Driver Code Ends

File: Difficulty: Basic/Last Digit of a number/last-digit-of-a-number.java
//{ Driver Code Starts

// } Driver Code Ends

// Back-end complete function Template for Java

import java.util.*;

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        
        // Read the input number
        int n = sc.nextInt();
        
        // Extract the last digit of the number
        int lastDigit = n % 10;
        
        // Get the absolute value of the last digit to handle negative numbers
        int absoluteLastDigit = (lastDigit < 0) ? -lastDigit : lastDigit;
        
        // Print the absolute value of the last digit
        System.out.println(absoluteLastDigit);
    }
}


//{ Driver Code Starts.
// } Driver Code Ends